UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Services/Interfaces/IFileShareService.cs

```diff
@@ -5,5 +5,6 @@ namespace UKHO.MaritimeSafetyInformation.Web.Services.Interfaces
     public interface IFileShareService
     {
         Task<IResult<BatchSearchResponse>> FssBatchSearchAsync(string searchText,string accessToken, string correlationId);
+        Task<IResult<BatchAttributesSearchResponse>> FssSearchAttributeAsync(string accessToken, string correlationId);
```

Comment:
Can we have acronyms capitalised please:
```suggestion
        Task<IResult<BatchSearchResponse>> FSSBatchSearchAsync(string searchText,string accessToken, string correlationId);
        Task<IResult<BatchAttributesSearchResponse>> FSSSearchAttributeAsync(string accessToken, string correlationId);
```

I think Martyn let the other one through while I was on holiday...but I'm back now 😃

Reply:
acronyms capitalised done